shared/src/packages/journal/libs/types/journal-entry-update-request-dto.type.ts

```diff
@@ -0,0 +1,10 @@
+type JournalEntryUpdateRequestDto = {
+  id: number;
+  userId?: number;
```

Comment:
we can't update user id
```suggestion
```

Reply:
I did it because I use this type for backend and frontend (in journal API). So for frontend I don't need userId, but in backend I need it, as for create and update  methods in service I need have one payload, so I can't pass it as second argument. So what can I do with it?

Reply:
then it should be separate types